www/js/Plugin/croppie.css

```diff
@@ -0,0 +1,179 @@
+.croppie-container {
+    width: 100%;
+    height: 100%;
+}
+.croppie-container .cr-image {
+	z-index: -1;
+	position: absolute;
+	top: 0;
+	left: 0;
+	transform-origin: 0 0;
+    max-width: none;
```

Comment:
This is 3rd party code from croppie library, should I correct someone's mistakes ?